packages/faency/src/components/InputTags.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { ChangeEvent, KeyboardEvent, ReactNode, useEffect, useState } from 'react'
+import { DismissableChip } from './DismissableChip'
+import { InputProps } from 'mdlz-prmtz'
+import styled from 'styled-components'
+import { theme } from '../theme'
+import { Box } from './Box'
+
+const StyledInput = styled('input')`
+  border: none;
+  background: none;
+  outline: none;
+  flex-grow: 1;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  max-height: ${theme.sizes[3]};
+  margin: 0 ${theme.space[1]} ${theme.space[1]} 0;
+`
+
+const Container = styled('div')<{ hasFocus: boolean }>`
+  position: relative;
+  padding: ${theme.space[1]} 0 0 ${theme.space[1]};
+  transition: box-shadow 360ms cubic-bezier(0.4, 0, 0.2, 1);
+  box-sizing: border-box;
+  border-radius: ${theme.radii[4]};
+  box-shadow: 0 0 0 1px ${theme.colors.gray};
+  display: flex;
+  flex-wrap: wrap;
+
+  ${({ hasFocus }) =>
+    hasFocus &&
+    `
+    box-shadow: 0 0 0 2px ${theme.colors.black};    
+  `}
+`
+
+const ChipsContainer = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+
+  &:first-child {
+    justify-items: flex-end;
+    max-width: 60%;
+  }
+
+  > * {
+    margin: 0 ${theme.space[1]} ${theme.space[1]} 0;
+  }
+`
+
+const SelectContainer = styled(Box)`
```

Comment:
On a UX/acceesibility perspective the popover should close clicking outside and as the select component having the ability to select the option with keyboard arrows.